README.md

```diff
@@ -54,6 +54,21 @@ docker container run -e GITLAB_PERSONAL_TOKEN=gitlabSampleToken -e GITLAB_PROJEC
 Reference gitlab repo: [generator test](https://gitlab.com/jackzhang/generator-test)
 
 
+## Options
+
+These can be specified using environment variables
```

Comment:
Nice! Thanks for the documentation 😀

Reply:
I added a more documentation on how to build and test local changes.